android/core/src/main/java/com/stadiamaps/ferrostar/core/NavigationViewModel.kt

```diff
@@ -99,18 +102,20 @@ class DefaultNavigationViewModel(
 ) : ViewModel(), NavigationViewModel {
 
   private var userLocation: UserLocation? = locationProvider.lastLocation
+  private var muteState: MutableStateFlow<Boolean?> =
+      MutableStateFlow(spokenInstructionObserver?.isMuted)
```

Comment:
There may be issues with this, but was curious since this is usually what you see:

```suggestion
      remember { mutableStateOf(spokenInstructionObserver?.isMuted) }
```